src/text_file_functions.cpp

```diff
@@ -514,8 +514,18 @@ QString AOApplication::get_court_sfx(QString p_identifier, QString p_misc)
 
 QString AOApplication::get_sfx_suffix(VPath sound_to_check)
 {
-  return get_real_suffixed_path(sound_to_check,
-                                {".opus", ".ogg", ".mp3", ".wav", ".mid", ".midi", ".xm", ".it", ".s3m", ".mod", ".mtm", ".umx" });
+  QStringList suffixes = {".opus", ".ogg", ".mp3", ".wav", ".mid", ".midi", ".xm", ".it", ".s3m", ".mod", ".mtm", ".umx" };
+  // Check if we were provided a direct filepath with a suffix already
+  QString path = sound_to_check.toQString();
+  // Loop through our suffixes
+  for (const QString &suffix : suffixes) {
+    // If our VPath ends with a valid suffix
+    if (path.endsWith(suffix, Qt::CaseInsensitive))
+      // Return that as the path
+      return get_real_path(sound_to_check);
+  }
+  // Otherwise, ignore the provided suffix and check our own
+  return get_real_path(sound_to_check, suffixes);
```

Comment:
This won't ignore the provided suffix as the comment states. It will add another suffix on top of the suffix already present. Is this what you were expecting?

Reply:
@oldmud0 This is accounted for due to how get_real_path is defined:
`   QString get_real_path(const VPath &vpath, const QStringList &suffixes={""});`
where "suffixes" has a suffix predefined as "nothing"

Reply:
@Crystalwarrior I mean, if you pass in `path/to/file.mp3` but it is actually a wav, it will try `path/to/file.mp3.wav` but not `path/to/file.wav`.